src/web_interface/templates/show_analysis.html

```diff
@@ -209,23 +206,10 @@ <h3>
                     <tr>
                         <td>
                             <div id="fileTreeAjax"></div>
-                            <script>
-                                $('#fileTreeAjax')
-                                    // generate links to the analysis page
-                                    .on("activate_node.jstree", function(e,data){window.location.href = data.node.a_attr.href;})
-                                    // generate file tree
-                                    .jstree({
-                                        "core" : {
-                                            'data' : {
-                                                'url' : function (node) {
-                                                    return node.id === '#' ?
-                                                        "/ajax_root/{{ uid | safe }}/{{ root_uid | safe }}" : "/ajax_tree/" + node["data"]["uid"] + "/{{ root_uid | safe }}";
-                                                }
-                                            }
-                                        },
-                                        "plugins" : [ "sort" ]
-                                    });
+                            <script> 
+                            const root_uid = "{{ root_uid | safe }}";
                             </script>
+                            <script src="{{ url_for('static', filename='js/show_analysis_fileTreeAjax.js') }}"></script>
```

Comment:
this can probably be moved to the top inside the `block head`. Also, the js file can probably merged with `start_compare.js` (and possibly `show_analysis_summary.js`) to a `show_analysis.js`.

Reply:
It doesn't work to move it to the head...